test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -186,6 +186,26 @@ Describe "Get-ChildItem" -Tags "CI" {
             $app.Target | Should -Not -Be $app.FullName
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Get-ChildItem.FileSystem provider.Wildcard matching behavior is the same between -Path and -Include parameters when using escape characters
```powershell
Expected exactly '/tmp/61481d6f-62f8-4289-9665-0772cb5e399c/a`[b]', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1: line 200
200:             $WithInclude.Name | Should -BeExactly $expectedPath
```